tests/framework/test_time.py

```diff
@@ -209,7 +234,7 @@ def test_empty_active_pop(SimulationContext, base_config, components):
     listener = [c for c in components if hasattr(c, "args") and "listener" in c.args][0]
     sim.setup()
     sim.initialize_simulants()
-    full_pop_index = sim.get_population().index
+    pop_index = full_pop_index(sim)
```

Comment:
why the rename from `full_pop_index` to `pop_index`? Regardless, it seems like the helper function should follow suit. 

nit: rename the helper function to `get_...()`